ocs_ci/ocs/ui/block_pool.py

```diff
@@ -173,3 +181,92 @@ def reach_pool_limit(self, replica, compression):
                     raise PoolStateIsUnknow(
                         f"pool {pool_name} is in unexpected state {pool_state}"
                     )
+
+    def pool_raw_capacity_loaded(self, pool_name):
+        """
+        Takes pool name and returns True if the raw capacity of the block pool is loaded
+        or returns False if the capacity is not loaded.
```

Comment:
What is laoded and not loaded means?

Reply:
raw capacity is loaded or not.
There is a section on the blockpool page that shows the raw capacity of the block pool.